core-services/release-controller/_releases/release-ocp-4.9.json

```diff
@@ -60,10 +60,18 @@
       "optional":true,
       "prowJob":{"name":"periodic-ci-openshift-release-master-nightly-4.9-e2e-gcp"}
     },
+    "gcp-techpreview":{
+      "optional":true,
+      "prowJob":{"name":"periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-techpreview"}
+    },
```

Comment:
Also do the same in `core-services/release-controller/_releases/release-ocp-4.10.json`